### PR TITLE
Create README-ja.md

### PR DESCRIPTION
**Pull request by yasuflatland-lf**
According to the request here, translated.
  Signed-off-by: Yasuyuki Takeo <yasuyuki.takeo@liferay.com>

## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Comment by ksatchit** [comment]
cc: @ToruMakabe

File: translations/README-ja.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+#### *他の言語は[ここを参照してください。](translations/TRANSLATIONS.md).*
+
+[🇰🇷](translations/README-ko.md) [🇨🇳](translations/README-chn.md)
+
+## Overview
+
+Litmusはクラウドネイティブのカオスエンジニアリングを行うためのツールセットです。Litmusは、Kubernetes上でカオスオーケストレーションするためのツールを提供し、SREがデプロイした環境の脆弱性を発見するのを支援します。SREはLitmusを使用して、最初はステージング環境でカオスエクスペリメントを実行し、最終的には本番環境でバグや脆弱性を発見するのに利用します。弱点を修正することで、システムの回復力を高めることを目指します。
+
+Litmusはクラウドネイティブなアプローチで、カオスの生成、管理、監視を行います。カオスは、以下のKubernetesカスタムリソース定義を使用してオーケストレーションされます。 (**CRDs**):
```
Comment (ToruMakabe) [inline translations/README-ja.md:26]: My suggestion: "カスタムリソース定義(**CRDs**)"

File: translations/README-ja.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+[![YouTube Channel](https://img.shields.io/badge/YouTube-Subscribe-red)](https://www.youtube.com/channel/UCa57PMqmz_j0wnteRa9nCaw)
+<br><br><br><br>
+
+#### *他の言語は[ここを参照してください。](translations/TRANSLATIONS.md).*
+
+[🇰🇷](translations/README-ko.md) [🇨🇳](translations/README-chn.md)
+
+## Overview
+
+Litmusはクラウドネイティブのカオスエンジニアリングを行うためのツールセットです。Litmusは、Kubernetes上でカオスオーケストレーションするためのツールを提供し、SREがデプロイした環境の脆弱性を発見するのを支援します。SREはLitmusを使用して、最初はステージング環境でカオスエクスペリメントを実行し、最終的には本番環境でバグや脆弱性を発見するのに利用します。弱点を修正することで、システムの回復力を高めることを目指します。
```
Comment (ToruMakabe) [inline translations/README-ja.md:24]: Could you please consider to unify the translation of "エクスペリメント/実験"? (regarding the whole document)

File: translations/README-ja.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+- **ChaosEngine**: KubernetesアプリケーションやKubernetesノードをChaosExperimentにリンクするためのリソースです。ChaosEngine は Litmus の Chaos-Operator によって監視され、次にChaos-Experimentsを実行します。
+- **ChaosExperiment**: カオス実験の設定パラメータをグループ化するためのリソースです。ChaosExperiment CRはChaosEngineによってエクスペリメントが起動されたときにオペレータによって作成されます。
+- **ChaosResult**: カオスエクスペリメントの結果を保持するリソース。Chaos-exporter は結果を読み込み、設定された Prometheus サーバにメトリクスをエクスポートします。
+
+カオスエクスペリメントは <a href="https://hub.litmuschaos.io" target="_blank">hub.litmuschaos.io</a> でホストされています。これはアプリケーション開発者やベンダーがカオスエクスペリメントを共有するハブであり、ユーザーが、運用中のアプリケーションの耐障害性を向上させるために使用します。
+
+![Litmus ワークフロー](/images/litmus-arch_1.png)
+
+## 利用事例
```
Comment (ToruMakabe) [inline translations/README-ja.md:36]: "ユースケース" maybe popular

File: translations/README-ja.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+[![IMAGE ALT TEXT](images/maxresdefault.jpg)](https://youtu.be/W5hmNbaYPfM)
+
+始めるには <a href="https://docs.litmuschaos.io/docs/next/getstarted.html" target="_blank">Litmusドキュメンテーション</a>を参照ください。。
+
+## Chaos Hubへの貢献
+
+<a href="https://github.com/litmuschaos/community-charts/blob/master/CONTRIBUTING.md" target="_blank">Chaos Hubに貢献するためのガイドライン</a>を参照ください。
+
+## アダプター
```
Comment (ToruMakabe) [inline translations/README-ja.md:52]: My suggestion: "採用ユーザー"

File: translations/README-ja.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+<a href="https://github.com/litmuschaos/litmus/blob/master/ADOPTERS.md" target="_blank">LitmusChaosのアダプター</a>を参照ください。
+
+(_カオスエンジニアリングのための実際にLitmusを使用している場合は、上記のページにPRを送信してください。_)
+
+## 考慮すべき事項
+
+Litmus（カオスフレームワークとして）で行う必要がある考慮すべき事項のいくつかは、ここに大まかにリストアップされています。これらの多くは、[ロードマップ](./ROADMAP.md)で述べられているように、すでに作業が行われています。特定の実験に関する詳細や制限については、それぞれの[実験ドキュメント](https://docs.litmuschaos.io/docs/pod-delete/)を参照してください。
+
+- Docker以外のコンテナランタイム（containerdなど）のネットワークカオス、CRIOはまだサポートされていません。
```
Comment (ToruMakabe) [inline translations/README-ja.md:62]: My suggestion: "Docker以外のコンテナランタイム（containerdやCRIOなど）のネットワークカオスはまだサポートされていません。"

In fact, litmus supports network chaos on containerd & CRI-O from [1.8.0](https://github.com/litmuschaos/litmus/releases/tag/1.8.0).  It's an issue of the original doc.

File: translations/README-ja.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+- Litmusのカオスオペレーターとカオスエクスペリメントはクラスタ内のkubernetesリソースとして動作します。エアギャップ環境の場合、カオスカスタムリソースとイメージをオンプレミスでホストする必要があります。
```
Comment (ToruMakabe) [inline translations/README-ja.md:63]: My suggestion: "エアギャップ環境" to "インターネットから隔離された環境"

File: translations/README-ja.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+- プラットフォーム固有のカオスエクスペリメント(AWSやGCPクラウド上での実験など)を実行しようとすると、アクセスの詳細はkubernetesのシークレットを介して渡されます。他のシークレット管理のモードのサポートはまだテスト/実装段階です。
+- カオスエクスペリメントの中には、エクスペリメントポッド内からDocker APIを利用しているものもあり、その場合はDockerソケットをマウントする必要があります。これらのエクスペリメントを実行するために、開発者/DevOps管理者/SRE のアクセスを許可するかどうかは、ユーザーに助言するようにします。
+- (まれに) カオスエスクペリメントで特権コンテナを利用する場合は、セキュリティポリシーを文書化することを推奨します。
```
Comment (ToruMakabe) [inline translations/README-ja.md:66]: My suggestion: "(まれに) カオスエスクペリメントで特権コンテナを利用する場合は" to "カオスエスクペリメントで特権コンテナを利用する(まれな)場合 は"

**Review by ToruMakabe — COMMENTED** [review]
Thanks @yasuflatland-lf ! I hope my suggestion is useful.

File: translations/README-ja.md (hunk 5 above)
Comment (yasuflatland-lf) [reply translations/README-ja.md:62]: What am I supposed to do here?

Comment (ToruMakabe) [reply translations/README-ja.md]: @ksatchit It seems that the original English doc still has [old info](https://github.com/litmuschaos/litmus#things-to-consider). litmus supports network chaos on containerd & CRI-O from 1.8.0. Could you check and consider to fix the original one?

Comment (ksatchit) [reply translations/README-ja.md]: Ah! Apologies. Thanks @ToruMakabe for pointing this out !! Fixing the original README.

Comment (ToruMakabe) [reply translations/README-ja.md]: @yasuflatland-lf The original doc has already been fixed. Please remove this line.

**Review by ToruMakabe — APPROVED** [review]
Thanks @yasuflatland-lf !

**Review by ksatchit — APPROVED** [review]

**Pull request merged**